cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/app/partitioned/PartitionExploreCorrector.java

```diff
@@ -41,7 +43,18 @@ public void configure() {
   /**
    * This worker can be used to bring a partitioned file set in sync with Explore. Run the worker with runtime
    * arguments: dataset.name=<name> [ verbose=<boolean> ] [ batch.size=<int> ] [ disable.explore=<boolean> ].
+   * <ul>
+   *   <li>dataset.name: The name of the partitioned file set to bring in sync. This argument is required.</li>
+   *   <li>verbose: If true, this will log a message for every partition; otherwise it will only log a report
+   *                of how many partitions were added or could not be added. Defaults to false.</li>
+   *   <li>disable.explore: If true, the worker will first disable and re-enable explore for the dataset, to
+   *                remove all existing partitions. If false, the worker will simply fail to add partitions
+   *                that already exist, but it will ignore these failures. Defaults to true.</li>
+   *   <li>batch.size: The number of transactions to add in a single transaction. Set this to a smaller number
```

Comment:
`batch.size` should be moved up so that the list of items matches the order of the parameters in the example (line 45)